webtau-graphql/src/main/java/org/testingisdocumenting/webtau/graphql/GraphQLCoverage.java

```diff
@@ -38,16 +40,41 @@ public void recordQuery(HttpValidationResult validationResult) {
         }
 
         Set<GraphQLQuery> graphQLQueries = schema.findQueries(validationResult.getRequestBody());
-        graphQLQueries.forEach(query -> coveredQueries.add(query, validationResult.getId(), validationResult.getElapsedTime()));
+        graphQLQueries.forEach(query -> coveredQueries.
+            add(query, validationResult.getId(), validationResult.getElapsedTime(), isErrorResult(validationResult)));
     }
 
-    Stream<GraphQLQuery> nonCoveredQueries() {
-        return schema.getSchemaDeclaredQueries().filter(o -> !coveredQueries.contains(o));
+    private boolean isErrorResult(HttpValidationResult validationResult) {
+      try {
+        return JsonUtils.convertToTree(JsonUtils.deserialize(validationResult.getResponse().getTextContent()))
+            .has("errors");
+      } catch (Exception e) {
+        e.printStackTrace();
```

Comment:
do we need to see this printed stack trace?

Reply:
I'd expect this to be a very rare occurrence, and then the detail might be useful.